src/test/java/htsjdk/variant/vcf/VCFCodec43FeaturesTest.java

```diff
@@ -0,0 +1,261 @@
+package htsjdk.variant.vcf;
+
+import htsjdk.samtools.util.CloseableIterator;
+import htsjdk.samtools.util.Interval;
+import htsjdk.samtools.util.TestUtil;
+import htsjdk.samtools.util.Tuple;
+import htsjdk.tribble.index.Index;
+import htsjdk.tribble.index.IndexFactory;
+import htsjdk.tribble.util.TabixUtils;
+import htsjdk.variant.VariantBaseTest;
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.variantcontext.VariantContextTestUtils;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+/**
+ **************************************************************************************************
+ * IMPORTANT NOTE: this class contains string constants that contain embedded non-ASCII characters
+ * used for testing VCF UTF-8 support. Edit with care.
+ **************************************************************************************************
+ */
+public class VCFCodec43FeaturesTest extends VariantBaseTest {
+    private static final String TEST_PATH = "src/test/resources/htsjdk/variant/vcf43/";
```

Comment:
make this `Path` e.g. `Paths.get(...)`

and below use `TEST_PATH.resolve(TEST_43_FILE)`

Reply:
Done.